--- cynergy/tests/test_register_multiple.py	2021-10-20 20:48:22.000000000 +0000
+++ cynergy/tests/test_register_multiple_processedpatch210.py	2021-12-09 12:02:26.501086074 +0000
@@ -1,3 +1,5 @@
+
+import warnings
 from typing import List
 
 from cynergy import container
@@ -35,6 +37,7 @@ class Main2(object):
 
 
 def test_register_multiple():
+    container.initialize(class_mapping={Example: Example2})
     container.register_many(Example, [Example1, Example2])
     instance = container.get(List[Example])
 
